misc/ballerina-maven-plugins/docerina-maven-plugin/src/main/java/org/ballerinalang/plugin/maven/doc/DocerinaMojo.java

```diff
@@ -84,6 +85,12 @@
 
     @Parameter(property = "outputZip", required = false)
     private String outputZip;
+    
+    /**
+     * Generates the table of content file for the package.
+     */
+    @Parameter(property = "generateToc", required = false, defaultValue = "false")
```

Comment:
shall we use generateTOC with all caps instead? TOC is what generally used.

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/9199/commits/da18aa7903275e6721a38078436f8455e87e944f